pkg/ctl/cmdutils/nodegroup_flags.go

```diff
@@ -45,7 +45,7 @@ func AddCommonCreateNodeGroupFlags(fs *pflag.FlagSet, cmd *Cmd, ng *api.NodeGrou
 
 	fs.StringSliceVar(&ng.SecurityGroups.AttachIDs, "node-security-groups", []string{}, "Attach additional security groups to nodes, so that it can be used to allow extra ingress/egress access from/to pods")
 
-	fs.StringToStringVar(&ng.Labels, "node-labels", nil, `Extra labels to add when registering the nodes in the nodegroup, e.g. "partition=backend,nodeclass=hugememory"`)
+	AddStringToStringVarPFlag(fs, &ng.Labels, "node-labels", "", nil, `Extra labels to add when registering the nodes in the nodegroup, e.g. "partition=backend,nodeclass=hugememory"`)
```

Comment:
also I think we can get rid of the additional examples in the specific descriptions (like "partition=...")

Reply:
Yeah that makes sense as k1=v1,k2=v2 is explanatory enough.